eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver/StaticSiteAccessConfigResolver.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\ConfigResolver;
+
+use eZ\Publish\Core\MVC\Exception\ParameterNotFoundException;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess\Provider\StaticSiteAccessProvider;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+/**
+ * @property-read \Symfony\Component\DependencyInjection\ContainerInterface $container
+ */
+class StaticSiteAccessConfigResolver extends SiteAccessConfigResolver
```

Comment:
Is it final or meant to be extended by EE SiteFactory db-based resolver?

Reply:
It is not a final class. I believe the reason is that it is a lazy service. It is not meant to be extended by EE SiteFactory DB-based resolver. It should had his own config resolver